docs/README.md

```diff
@@ -19,7 +19,7 @@ Kontena's key features include:
 * Log and statistics aggregation with streaming
 * Access control and roles for Kontena users
 
-Kontena is administered via the Kontena command line interface, **Kontena CLI**. At the moment, there is no graphical UI for Kontena.
+Kontena is administered via the Kontena command line interface, **Kontena CLI**. **[Kontena Cloud](https://cloud.kontena.io)** provides hosted Graphical UI for monitoring, analyzing and operating Kontena Platforms.
```

Comment:
"provides `a` hosted.".

maybe also "Graphical UI" -> "(?graphical) user interface".

Also looks like `command-line interface` comes with a hyphen, for example in (Wikipedia)[https://en.wikipedia.org/wiki/Command-line_interface]